src/api/Feed.js

```diff
@@ -162,6 +162,56 @@ class Feed extends Base {
         this.updateFeedItem({ ...annotation, isPending: false }, id);
     }
 
+    updateAnnotation = (
+        file: BoxItem,
+        annotationId: string,
+        text: string,
+        permissions: AnnotationPermission,
+        successCallback: (annotation: Annotation) => void,
+        errorCallback: ErrorCallback,
+    ): void => {
+        const annotationData = {
+            message: text,
+        };
+
+        this.annotationsAPI = new AnnotationsAPI(this.options);
+
+        if (!file.id) {
+            throw getBadItemError();
+        }
+
+        this.file = file;
+        this.errorCallback = errorCallback;
+
+        // Create action has completed, so update the existing pending item
+        this.updateFeedItem({ ...annotationData, isPending: true }, annotationId);
```

Comment:
Is the `annotationData` variable needed or can the patch be done inline here?

Reply:
It can be done inline.